lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -975,8 +977,7 @@ def setUp(self):
         self.user = UserFactory.create()
         self.assertTrue(self.client.login(username=self.user.username, password='test'))
 
-        self.time_with_tz = "due Sep 18, 2013 at 11:30 UTC"
-        self.time_without_tz = "due Sep 18, 2013 at 11:30"
+        self.time_with_tz = "2013-09-18 11:30:00+00:00"
```

Comment:
I'd like to understand what changes are being made in the UI here.  Changing the output in the test smells like a possible undesired change.